workspaces/liferay-customer-workspace/client-extensions/liferay-customer-global-css/client-extension.yaml

```diff
@@ -1,6 +1,6 @@
 assemble:
     - from: assets
-      into: static
+        into: static
```

Comment:
@amosfong this extra two spaces seems to break the deploy

Reply:
oh my formatsource did that, I'll revert it